src/renderer/components/ft-profile-selector/ft-profile-selector.js

```diff
@@ -35,6 +35,15 @@ export default Vue.extend({
       return this.profileList.map((profile) => {
         return profile?.name?.length > 0 ? Array.from(profile.name)[0].toUpperCase() : ''
       })
+    },
+    activeHasProfileImage: function () {
+      // the profile image url should both exist and be a non-empty string
+      return this?.activeProfile?.imageUrl && this.activeProfile.imageUrl?.length > 0
```

Comment:
Please remove unnecessary optional chaining operators, `this` will always have a value
```suggestion
      return this.activeProfile?.imageUrl && this.activeProfile.imageUrl?.length > 0
```